### PR TITLE
Allow single file change without CLA

### PR DESCRIPTION
**Pull request by aaime**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
-Contributors are asked to provide a Contributor License Agreement for working on the project:
+When submitting pull request:
 
-* [Corporate contributor license](https://www.osgeo.org/resources/corporate-contributor-license/)
-* [Individual contributor license](https://www.osgeo.org/resources/individual-contributor-license/)
+* **Small Contribution / Single Source Code File:** For a small change to a single source file a project committer can review and apply the change on your behalf. This is a quick workaround allowing us to correct spelling mistakes in the documentation, clarify a javadoc, or accept a very small fix.
+
+  We understand that fixing a single source file may require changes to several test case files to verify the fix addresses its intended problem.
+
+* **Large Contributions / Multiple Files / New Files:** To  contribute a new file, or if your change effects several files, sign a [Code Contribution License](http://docs.geotools.org/latest/developer/procedures/contribution_license.html). It does not take long and you can send it via email.
```
Comment (bradh) [inline CONTRIBUTING.md]: Should the link go to http://docs.geoserver.org/latest/en/developer/policies/committing.html instead of to GeoTools?

**Comment by aaime** [comment]
Fixed the link

**Comment by bradh** [comment]
@aaime Do you still need this branch?

**Comment by aaime** [comment]
Nope, removed

**Pull request merged**